### ISSUE TITLE
solver.py tensor `c_org_test` no `2` and `3` values

### ISSUE THREAD

**Issue by annhit29** [tdn02007/UC-GAN-Unified-cipher-generative-adversarial-network#1]
Hi, your UC-GAN model is really interesting and I'm using your model. I am trying to understand your code, but it seems like in solver.py, there's never `2` or `3` in `c_org_test`:
c_org_test = tensor([0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1]) #this is the printing value of `c_org_test` 

From line 422 to line428:
```python
            with torch.no_grad():

                #### for testline ##############
                data_loader_test = self.data_loader_test

                data_iter_test = iter(data_loader_test)
                x_fixed_test, c_org_test = next(data_iter_test)
```
However, we need `2` and `3` for `id2` and `id3`, otherwise we'll always stay in the while loop to "Fix batch again" and "Fix batch again done", and `iden` is never equal to `1`.

How should I do to get `2` and `3` in `c_org_test` ?

Btw, which pytorch version do you use for this model? (Mine is 2.1.0+cu121.) I'm asking this question because it's possible that a different pytorch version of `next` function produces this problem.

Thank you in advance for your reply :D

**Comment by annhit29** [comment]
Update: 
For the discriminator: there is another problem: when I print the argument of line `out_cls` and `label_org` 347 `d_loss_cls = self.classification_loss(out_cls, label_org)`, why are there 4 labels in `label_org` of the discriminator instead 2 (0 = false, 1 = true for example)?
